jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+	preset: '@automattic/calypso-build',
```

Comment:
I don't see this dependency listed. Is this working as expected? Did this config need to be included?

Reply:
Thanks for pointing that out!

The test task is running fine and there are no warning or errors, but that actually might have been an oversight from my part and it's probably using the vanilla jest configuration... the actual test is quite simple so it doesn't really need anything from this preset, although I think this is the preset we should use to align with the Calypso standards, AFAIK. 

I'll double-check this now and add the dependency if needed.

Reply:
The testing via wp-scripts shouldn't need any dependency changes. It would be good to remove this file, the lock changes, and the dependencies changes.

We shouldn't need to add any dependencies here.

Reply:
Copy that, thanks for letting me know! I'll clean this up and just use `wp-scripts`. I'll do the same for `wpcom-blocks` since it has a similar setup.